src/core/dfn-finder.js

```diff
@@ -182,7 +182,10 @@ export function decorateDfn(dfnElem, idlAst, parent, name) {
     const lCaseParent = parent.toLowerCase();
     const middle = lCaseParent ? `${lCaseParent}-` : "";
     let last = name.toLowerCase().replace(/[()]/g, "").replace(/\s/g, "-");
-    if (last === "") last = "the-empty-string";
+    if (last === "") {
+      last = "the-empty-string";
+      dfnElem.setAttribute("aria-label", "the empty string");
```

Comment:
Nit: gotta reduce that bundle size 😄
```suggestion
      dfnElem.setAttribute("aria-label", last);
```

Reply:
"the-empty-string" is different to "the empty string", so will keep this one.

Reply:
Ah. Missed that.